configure.py

```diff
@@ -447,6 +447,7 @@ def find_ninja():
     'test/boost/UUID_test',
     'test/boost/advanced_rpc_compressor_test',
     'test/boost/allocation_strategy_test',
+    'test/boost/alternator_expressions_test',
```

Comment:
I think you should consider putting the new tests inside the existing alternator_unit_test instead of creating a separate source file. alternator_unit_test is one of the smallest unit test source files we have, there's no risk it's becoming too big. In the past we used to create a separate executable for each test which made it very expensive to have many small test files. I don't know if that's true any more, but in any case I don't see a benefit of splitting a file that developers will normally want to run in its entirety anyway.